scripts/tempesta.sh

```diff
@@ -329,18 +323,19 @@ start_tfw_logger()
 			sysctl -e -w net.tempesta.state=stop
 			unload_modules
 			tfw_irqbalance_revert
-			error "tfw_logger failed to start, see $mmap_log for details"
+			error "tfw_logger failed to start, see logs for details"
 		fi
 
 		sleep 0.1
 	done
-
 }
 
 stop_tfw_logger()
 {
 	if [ -e $tfw_logger_pid_path ]; then
-		"$utils_path/tfw_logger" -s
+		echo "...stopping tfw_logger"
+		"$logger_path/tfw_logger" --stop || \
```

Comment:
For some reason you removed `stop_daemon()` as well as parsing `-s` or `--stop` options, so this seems not working anymore